components/chef-run/lib/chef-run/config.rb

```diff
@@ -80,16 +80,6 @@ def exist?
         File.exist? location
       end
 
-      def create_directory_tree
-        FileUtils.mkdir_p(File.dirname(default_location))
-        FileUtils.mkdir_p(File.dirname(stack_trace_path))
-        FileUtils.mkdir_p(telemetry_path)
-      end
-
-      def create_default_config_file
```

Comment:
Moved behavior to Startup,   because a Configuration shouldn't be altering your file system.